lib/components/narrative/metro/departure-times-list.tsx

```diff
@@ -27,10 +24,7 @@ export const DepartureTimesList = (props: DepartureTimesProps): JSX.Element => {
     return (
       <button
         className={isRealTime ? 'realtime active' : 'active'}
-        title={`${intl.formatMessage(
-          { id: 'components.MetroUI.arriveAtTime' },
```

Comment:
Why was this removed?

Reply:
Because it replaces the departure time, creating inconsistencies for screenreaders: 
![image](https://user-images.githubusercontent.com/115499534/222815346-1a9dbe72-133d-4fa2-9a41-415925e5d0c6.png)

Reply:
Doesn't aria-label override `title`? I don't like that you hover over the text and the title contains the exact same text

Reply:
The title should actually contain the time and also whether or not the departure is realtime, so in some cases it will be different! 

I can add an aria label as well! I think it should probably have both, you're right.

Reply:
I went ahead and added the "based on schedule data" string to scheduled trips as well, so that the title is never the same text as the button!

Reply:
I like this more! Would it be possible to re-add the arrives by info? That is a helpful piece of information

Reply:
I agree the arrives by info is helpful, but I don't think putting it as the title of this button feels like a super intuitive place for it. May be worth having a larger team discussion for the best place to present that info within the itinerary?

Reply:
This decision came out of a team discussion on this issue a while ago. I agree it's not perfect but would prefer the info in the title to it not being there at all!